packages/client/TODO.md

```diff
@@ -84,10 +84,10 @@ Client manages the token lifecycle and storage
 - [AM] client.profile.createSetFollowNFTUriTypedData(request)
 - [APQ] client.profile.pendingApprovalFollows(request)
 
-- [Q] client.profile.doesFollow(request)
-- [PQ] client.profile(s).following({ address })
-- [PQ] client.profile.fetchFollowers({ profileId })
-- [Q] client.profile.followerNftOwnedTokenIds({ address, profileId })
+- [x] client.profile.doesFollow(request)
+- [x] client.profile(s).allFollowing({ address })
```

Comment:
for the future leave the annotations in [] like [PQ], which will tell us which methods require auth or pagination etc.